Source/Core/Core/HotkeyManager.cpp

```diff
@@ -237,6 +239,43 @@ bool IsPressed(int id, bool held)
   return false;
 }
 
+// This function exists to load the old "Keys" group so pre-existing configs don't break.
+// TODO: Remove this at a future date when we're confident most configs are migrated.
+static void LoadLegacyConfig(ControllerEmu::EmulatedController* controller)
```

Comment:
Can we / should we remove the old section from the configuration?

Reply:
No I think we should leave it there to keep the config functional on old Dolphin versions.